### DISCUSSION TITLE
Yarn error `No such file or directory: 'install'`

### DISCUSSION THREAD

**Discussion by ghost** [zwave-js/zwave-js-ui#2745] [Q&A] [answered]
### Checklist

- [ ]  I am not using Home Assistant. Or: a developer has told me to come here.
- [ ] I have checked the troubleshooting section and my problem is not described there.
- [ ] I have read the changelog and my problem is not mentioned there.

### Deploy method

Manually built (git clone - yarn build - yarn install

### Z-Wave JS UI version

yarn via git latest

### ZwaveJS version

latest

### Describe the bug

Simple:
following bear metal instructions:
yarn install
00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'install'

### To Reproduce

git clone https://github.com/zwave-js/zwave-js-ui
cd zwave-js-ui
yarn install
yarn run build
yarn start

### Expected behavior

install properly

**Comment by robertsLando** [comment]
Do you have nodejs npm and yarn installed on your system?

**Comment by ghost** [comment]
uzwave@HA-1:~$ npm -v
8.19.2
yarn --version
0.32+git

followed:
git clone https://github.com/zwave-js/zwave-js-ui
cd zwave-js-ui
yarn install
yarn run build
yarn start

yarn install  Error: no install

**Comment by robertsLando** [comment]
Try using `yarn` without `install` as first command. Anyway it's working on my side and never seen such error

**Comment by ghost** [comment]
Thanks for great  work!
Now Yarn needs some scenario to execute...
install or build or any scenario do not exist  can't be done
is this right repo:
git clone https://github.com/zwave-js/zwave-js-ui
this is waht I get after clone 

app.ts  CHANGELOG.md  docs        kustomization.yaml  nodemon.json  pkg          snippets  store                 tsconfig.json  yarn.lock
bin     config        hass        lib                 package.json  README.md    src       test                  views
build   docker        kubernetes  LICENSE             package.sh    SECURITY.md  static    tsconfig.eslint.json  wallaby.js

**Comment by robertsLando** [comment, marked as answer]
@catchthemonster I think you haven't installed yarn correctly. The version should be:

```
$ yarn --version
3.2.1
```

**Comment by ghost** [comment]
Daniel, you are right!
I removed yarn installed by debian adn npm remove yarn

i ran npm install --global yarn 
in zwave-js-ui 
yarn
yarn build
yarn install
yarn start
and viola everything is back online ...
Thanks, sorry for the hassle ...

**Comment by robertsLando** [comment]
No worries, Glad you fiexed your problem :)
